vendor/phantomjs-launcher.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+var system = require('system')
+var fs = require('fs')
+var page = require('webpage').create()
+var url = system.args[1]
+page.viewportSize = {
+  width: 1024,
+  height: 768
+}
```

Comment:
@sglanzer Those are just the size of the `window`. The multiple size thing changes the width/height of the `.ember-testing-container`